src/manual/peer2peer/extension/background.js

```diff
@@ -44,3 +48,8 @@ function cancelScreenSharing() {
     chrome.desktopCapture.cancelChooseDesktopMedia(desktopMediaRequestId);
   }
 }
+
+function getChromeVersion() {
+  var raw = navigator.userAgent.match(/Chrome\/([0-9]+)\./);
```

Comment:
Btw we could also update the browser shim adapter.js which has a proven browser detection mechanism (including version extraction)? E.g. this currently does not match Chromium.